modules/manage/partials/monitor-connectors.adoc

```diff
@@ -313,6 +313,59 @@ producers.
 
 Rate of producer errors when producing records to Redpanda.
 
+== Connectors Issues Handling
+
+Redpanda’s Support team monitor Managed Connectors 24/7 to ensure they meet service level agreements (SLAs). If incidents occur, teams at Redpanda trigger an incident response process to quickly mitigate them.
+
+=== Increasing Consumer lag
```

Comment:
I think we can also:
- export new dashboard with the new lag rate metric
- describe the lag rate metric
- also export alert for the metric, if possible